assurance/execution.test.js

```diff
@@ -0,0 +1,87 @@
+const { execute } = require('../test/helpers/execution');
+const { mapValueSets } = require('../test/helpers/valueSetMapper');
+const { loadELM, loadJSONFixture, loadValueSets } = require('../test/helpers/fixtureLoader');
+
+let valueSetMap;
+let elm;
+let patientBundle;
+
+beforeEach(() => {
+  // Set up necessary data for cql-execution
+  valueSetMap = {
+    SteamVS: {
+      1: [
+        {
+          code: 'example-code',
+          system: 'www.example.com',
+        },
+      ],
+    },
+  };
+  elm = loadJSONFixture(__dirname, 'fixtures/elm/testLib.json');
+  patientBundle = loadJSONFixture(__dirname, './fixtures/patients/test-patient-1.json');
```

Comment:
Do we need a `beforeEach` here? Do these values get changed by the blocks of testing code? If not, I believe we can use `beforeAll` to avoid loading fixtures anew before each test.

Reply:
The valueSetMap is changed in only one of the unit tests, so  a `beforeAll` is probably all that's needed here. It'd be less costly to just initialize a new `valueSetMap` in that test specifically.